CONTRIBUTING.rst

```diff
@@ -228,3 +227,20 @@ To run tests in parallel on eight processors::
 To run tests verbosely, displaying all print statements to terminal::
 
     py.test  -vvs
+
+Things to Keep in Mind When Contributing
+----------------------------------------
+
+#. An update to the ``CHANGELOG`` is required if any important changes are made that needs to be communicated such as:
+
+   * Changes in the API.
+   
+   * Addition or deletion of CLI options.
+   
+   * Addition of any new feature to the program.
+   
+   * Any other miscellaneous changes to the program.
+   
+#. If there is a code change, a significant document, or any other changes, you must update the ``AUTHORS`` to include your own name.
+
+#. A link to the related documentation, which contains a more detailed guide for contributing code or documentation.
```

Comment:
Same, include this between points 3 and 4, and this time actually include the link to the documentation for contributing code: https://scancode-toolkit.readthedocs.io/en/stable/contribute/contrib_dev.html

Reply:
Added. Can you look into the changes?